### ISSUE TITLE
Plugin is not measuring number of lines of Kotlin code

### ISSUE THREAD

**Issue by sidd9433** [arturbosch/sonar-kotlin#30]
After the analysis it can get the kotlin issue in sonar but it cant get the number of lines of kotlin code.

Its a mixed project, with java & kotlin.

![image](https://user-images.githubusercontent.com/20203936/30797802-93fe0182-a1d7-11e7-87af-a160afbd326e.png)

Here is the configuration in build.gradle

```
sonarqube {
    properties {
		property "sonar.host.url", "XXXXXX"
		property "sonar.login", "XXXXXX"
		property "sonar.projectKey", "myproj-android"
		property "sonar.projectName", "myproj-android"
		property "sonar.java.source", VERSION_1_7
		property "sonar.verbose", "true"
		property "sonar.showProfiling", "true"
		property "sonar.sources", "src/main"
		property "sonar.projectVersion", "1.0.0"
		property "sonar.java.coveragePlugin", "jacoco"
		property "sonar.junit.reportsPath", "build/test-results/XXXX"
		property "sonar.jacoco.reportPath", "build/jacoco/XXXX.exec"
		property "sonar.java.binaries", "build/intermediates/classes/XXXX/XXXX"
		property "sonar.android.lint.report", "build/reports/lint-results.xml"
	}
}
```
Please help

**Comment by arturbosch** [comment]
Have you built from the source/master branch? It is not released at. If your using 0.2.3 it is not there.

**Comment by sidd9433** [comment]
I'll try that

**Comment by sidd9433** [comment]
@arturbosch Indeed its working from the build from master. Any idea when will be released ?

**Comment by sidd9433** [comment]
@arturbosch Thanks, will wait for that.

**Comment by arturbosch** [comment]
I released `0.2.4` with basic metrics and jacoco support!
